parsl/providers/condor/condor.py

```diff
@@ -213,15 +217,16 @@ def submit(self, command, blocksize, tasks_per_node, job_name="parsl.auto"):
             f.write(job_config["worker_init"] + '\n' + wrapped_command)
 
         user_script_path = self.channel.push_file(userscript_path, self.channel.script_dir)
-        job_config["input_files"] = user_script_path
+        the_input_files = [user_script_path] + self.transfer_input_files
+        job_config["input_files"] = ','.join(the_input_files)
         job_config["job_script"] = os.path.basename(user_script_path)
 
         # Construct and move the submit script
         self._write_submit_script(template_string, script_path, job_name, job_config)
         channel_script_path = self.channel.push_file(script_path, self.channel.script_dir)
 
         cmd = "condor_submit {0}".format(channel_script_path)
-        retcode, stdout, stderr = super().execute_wait(cmd, 3)
+        retcode, stdout, stderr = super().execute_wait(cmd, 10)
```

Comment:
Is 10 a high enough number? This needs to be higher than the plausible execution time for condor submit, and 10 seconds for a batch job submission doesn't sound implausible.

Reply:
Should be configurable parameter exposed to user, to be honest. 
Suggest a different PR tackles that, and patch it to 30s here?

Reply:
Yes this has been opened in #891

Reply:
(so agree that a different PR should do it)